examples/bucket_setup_using_autokey/main.tf

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "autokey" {
+  source = "../../modules/autokey"
```

Comment:
same here
```suggestion
  source  = "terraform-google-modules/terraform-google-kms/google//modules/autokey"
  version = "~> 4.0"
```

Reply:
after updating as your suggestion i am seeing module not found error

```
 
│ Module "autokey" (from ../../../examples/autokey_setup/main.tf:17) cannot
│ be found in the module registry at registry.terraform.io.

```

Shall I update the source+version after merging this pr for examples? or is there any workaround?